### ISSUE TITLE
Pass argument pretrained_dir in build_refinenet to frontend_builder

### ISSUE THREAD

**Issue by msseibel** [GeorgeSeif/Semantic-Segmentation-Suite#109]
Please fill out this issue template before submitting. **Issues which do not fill out this template, or are already answered in the FAQs will simply be closed.**

**Please go to Stack Overflow for help and support.** Also check past issues as many are repeats. Also check out the Frequently Asked Questions (FAQs) below in case your question has already been answered in an issue!

Issues should be one of the following:

1. Feature requests
2. Bug reports

------------------------

## Please specify the following information when submitting an issue:
- **What are your command line arguments?:** None
- **Have you written any custom code?**:  No
- **What have you done to try and solve this issue?:**  Nothing yet
- **TensorFlow version?**:  Not relevant

### Describe the problem

Something between bug and feature:
See the linked line below:
https://github.com/GeorgeSeif/Semantic-Segmentation-Suite/blob/4b7eb0c1a1622367a4378cfd42d3b5b621bcb3e8/models/RefineNet.py#L158
The argument `pretrained_dir` from the method `build_refinenet` is not passed towards the method `frontend_builder.build_frontend` and therefore currently useless.  

The argument `pretrained_dir` is also not passed in 
https://github.com/GeorgeSeif/Semantic-Segmentation-Suite/blob/4b7eb0c1a1622367a4378cfd42d3b5b621bcb3e8/models/DeepLabV3_plus.py#L75 

Are there any reasons for not passing it?

My suggestion:
`logits, end_points, frontend_scope, init_fn  = frontend_builder.build_frontend(inputs, frontend, is_training=is_training,pretrained_dir=pretrained_dir)`
 

### FAQs

- **Question:** I got an `InvalidArgumentError` saying that `Dimensions of inputs should match` **Answer:** See issue #17

- **Question:** Can you upload pre-trained weights for these networks? **Answer:** See issue #57

- **Question:** Do I need a GPU to train these models? **Answer:** Technically no, but I'd highly recommend it. I was able to train the models pretty well in about a day using a 1080Ti GPU. Training on CPU would take much longer than that.

- **Question:** Will you be adding the FCN or U-Net models? **Answer:** No I won't be adding those simply because they're a few years old and state-of-the-art has moved past that.

- **Question:** I got an invalid argument error when using the InceptionV4 model. Am I doing something wrong? **Answer:** No you're not! Due to the design of the InceptiveV4 model, when you end up upsampling you do some rounding which creates a shape mismatch. _This only happens when you end up having to use the `end_points['pool5']`_. See the code for some of the models if you want to check whether the model will use `end_points['pool5']`.
